Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Sharing/SpatialCoordinateSystemParticipant.cs

```diff
@@ -6,267 +6,359 @@
 using Microsoft.MixedReality.Toolkit.Utilities;
 using System;
 using System.IO;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using UnityEngine;
+using UnityEngine.XR.WSA;
 
 namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
 {
     /// <summary>
     /// The SpectatorView helper class for managing a participant in the spatial coordinate system
     /// </summary>
-    internal class SpatialCoordinateSystemParticipant : DisposableBase
+    internal class SpatialCoordinateSystemParticipant : MonoBehaviour
```

Comment:
>MonoBehaviour [](start = 56, length = 13)

Does this really need to be a monobehaviour? Settings transforms for this component every update doesn't really seem needed. Or at least, that wasn't the mechanism for using this transform before and it feels a bit unclear why its needed here now.

Reply:
The reason I made this a MonoBehaviour was to get an update loop so that the state message can be checked every frame. For a WorldAnchor-based coordinate, there's unfortunately no event to indicate that the WorldAnchor needed to modify the transform's position and rotation, so the only way to detect those changes is to poll each frame.

As I mentioned in the other comment, the change message is only sent if it's different from the previous frame's message, so that should minimize network traffic from these status updates.